doc/panes/editor.rst

```diff
@@ -38,6 +38,14 @@ You can separate cells by lines starting with either:
 * ``# <codecell>`` (IPython notebook cell separator)
 
 Providing a description to the right of the separator will give that cell its own name in the :guilabel:`Outline Explorer`.
+You can also create "subsections" by adding ``%`` signs to the cell separator, e.g. ``# %%%`` to create a level 2 subsection, ``# %%%%`` for level 3, etc...
```

Comment:
```suggestion
You can also create "subsections" by adding more ``%`` signs to the cell separator, e.g. ``# %%%`` to create a level 2 subsection, ``# %%%%`` for level 3, etc.
```

* Add clarifying "more"
* Tweak ending punctuation so sentence ends in a period.